molecule/test/unit/driver/test_delegated.py

```diff
@@ -24,6 +24,7 @@
 
 from molecule import config
 from molecule.driver import delegated
+from ...conftest import is_subset
```

Comment:
is this intended to be

``` python
from molecule.test.conftest import is_subset
```

Reply:
indeed. I wonder why it passed flake8, if I remember well it was able to catch suff like this.